packages/create/templates/hello-drizzle/src/operations.ts

```diff
@@ -0,0 +1,33 @@
+import { HandlerContext, TransactionContext, Transaction, GetApi, ArgSource, ArgSources } from '@dbos-inc/dbos-sdk';
+import { DBOSHello } from './schema';
+import { sql } from 'drizzle-orm';
+import { NodePgDatabase } from 'drizzle-orm/node-postgres';
+
+export class Hello {
+
+  @GetApi('/') // Serve a quick readme for the app
+  static async readme(_ctxt: HandlerContext) {
+    const readme = '<html><body><p>' +
+           'Welcome to the DBOS Hello App!<br><br>' +
+           'Visit the route /greeting/:name to be greeted!<br>' +
+           'For example, visit <a href="/greeting/dbos">/greeting/dbos</a>.<br>' +
+           'The counter increments with each page visit.<br>' +
+           'If you visit a new name like <a href="/greeting/alice">/greeting/alice</a>, the counter starts at 1.' +
+           '</p></body></html>';
+    return Promise.resolve(readme);
+  }
+
+  @GetApi('/greeting/:user') // Serve this function from HTTP GET requests to the /greeting endpoint with 'user' as a path parameter
+  @Transaction()  // Run this function as a database transaction
+  static async helloTransaction(ctxt: TransactionContext<NodePgDatabase>, @ArgSource(ArgSources.URL) user: string) {
+    const output = await ctxt.client.execute(sql`
```

Comment:
Wow the syntax is interesting. I think our SQL injection detection will mark this as string concatenation...

Reply:
That didn't quite end up the way I wanted needs work.  He keeps saying "literal" when it isn't literal, and also isn't what we want, which is "sanitized", of which any compile-time constant would be, but so would anything interpolated by a trusted library.

Reply:
Does Drizzle sanity check values like `${user}`? Does Drizzle turn this huge string into a parametrized statement?

Reply:
Yes, the `sql` is what does it.  Drizzle has the option of making it a parameterized statement or not, depending on what makes sense.  (Often it does not make sense to do a parameterized statement because that is more DB round trips in postgres, Prepare / Bind / Execute instead of Query.)